index.js

```diff
@@ -30,7 +30,9 @@ function CDNUp(bucket, options) {
   this.check = options.check;
   this.bucket = bucket;
   this.client = pkgcloud.storage.createClient(options.pkgcloud || {});
-  this.acl = options.acl || 'public-read';
+  if ('acl' in options) {
+    this.acl = options.acl || 'public-read';
```

Comment:
IMO, having a default here just makes this confusing. The only way to get to the default now is passing a falsey value for `options.acl` which seems like it should just not set it rather than defaulting.